dipy/reconst/dti.py

```diff
@@ -1900,12 +1894,19 @@ def nlls_fit_tensor(
 
 @warning_for_keywords()
 def restore_fit_tensor(
-    design_matrix, data, *, sigma=None, jac=True, return_S0_hat=False, fail_is_nan=False
+    design_matrix,
+    data,
+    *,
+    sigma=None,
+    jac=True,
+    return_S0_hat=False,
+    fail_is_nan=False
 ):
     """
-    Use the RESTORE algorithm to calculate a robust tensor fit.
-
-    See :footcite:p:`Chang2005` for further details about the method.
+    Use the RESTORE algorithm [1]_ to calculate a robust tensor fit.
+    Note that [1]_ does not define Geman–McClure M-estimator weights as
+    claimed (instead, Cauchy M-estimator weights are defined), but this
+    function does define these weights correctly.
```

Comment:
please, keep the `footcite`

Reply:
pretty sure this was all caused by rebase issues.... there were simply so many rebases, that I must have lost track. I will put them back

Reply:
sorted, added Chang2012 to the references as well